modules/bmp/image_loader_bmp.cpp

```diff
@@ -91,11 +91,13 @@ Error ImageLoaderBMP::convert_to_image(Ref<Image> p_image,
 		// the data width in case of 8/4/1 bit images
 		const uint32_t w = bits_per_pixel >= 24 ? width : width_bytes;
 		const uint8_t *line = p_buffer + (line_width * (height - 1));
+		const uint8_t *end_buffer = p_buffer + p_header.bmp_info_header.bmp_size_image;
```

Comment:
bmp_size_image do we trust this number? How is this calculated? I would expect the buffer size to be calculable from the size of the bmp.

Reply:
I found that buffer size is calculated from `bmp_file_size - bmp_file_offset`. However, if two variables are corrupt but the buffer had allocated as calculated. I think it may be safe to trust this calculation.